cmd/kubelet/app/options/options.go

```diff
@@ -34,16 +34,20 @@ import (
 type KubeletServer struct {
 	componentconfig.KubeletConfiguration
 
-	AuthPath      util.StringFlag // Deprecated -- use KubeConfig instead
-	KubeConfig    util.StringFlag
-	APIServerList []string
-
-	RunOnce bool
+	AuthPath   util.StringFlag // Deprecated -- use KubeConfig instead
+	KubeConfig util.StringFlag
 
 	// Insert a probability of random errors during calls to the master.
 	ChaosChance float64
 	// Crash immediately, rather than eating panics.
 	ReallyCrashForTesting bool
+
+	// TODO(mtaufen): It is increasingly looking like nobody actually uses the
+	//                Kubelet's runonce mode anymore, so it may be a candidate
+	//                for deprecation and removal.
```

Comment:
We were planning on using it for self hosting bootstrap, is that not the case (I didn't look through all the comments, apologies if this has already been discussed).